canonical-websites/partners.ubuntu.com
# Update to links https

## Done

- Update all non https links
- Add ASSET_SERVER_URL variable to project and update v1 links with this var

## QA

- `./run`
- [0.0.0.0:8003](http://0.0.0.0:8003/)
- Check that all links still work ...

## Issue / Card

- Fixes https://github.com/canonical-websites/partners.ubuntu.com/issues/127
- Fixes https://github.com/canonical-websites/partners.ubuntu.com/issues/128
- Opened https://github.com/canonical-websites/partners.ubuntu.com/issues/142
